lib/rules/eol-last.js

```diff
@@ -1,62 +1,94 @@
 /**
- * @fileoverview Require file to end with single newline.
+ * @fileoverview Require or disallow newline at the end of files
  * @author Nodeca Team <https://github.com/nodeca>
  */
 "use strict";
 
 //------------------------------------------------------------------------------
+// Requirements
+//------------------------------------------------------------------------------
+
+const lodash = require("lodash");
+
+//------------------------------------------------------------------------------
 // Rule Definition
 //------------------------------------------------------------------------------
 
 module.exports = {
     meta: {
         docs: {
-            description: "enforce at least one newline at the end of files",
+            description: "require or disallow newline at the end of files",
             category: "Stylistic Issues",
             recommended: false
         },
-
         fixable: "whitespace",
-
         schema: [
             {
-                enum: ["unix", "windows"]
+                enum: ["always", "never", "unix", "windows"]
             }
         ]
     },
-
     create(context) {
 
         //--------------------------------------------------------------------------
         // Public
         //--------------------------------------------------------------------------
 
         return {
-
             Program: function checkBadEOF(node) {
-
                 const sourceCode = context.getSourceCode(),
                     src = sourceCode.getText(),
-                    location = {column: 1},
-                    linebreakStyle = context.options[0] || "unix",
-                    linebreak = linebreakStyle === "unix" ? "\n" : "\r\n";
+                    location = {
+                        column: 1,
```

Comment:
Good catch, and yeah, `line` is one-based and `column` is zero-based: http://eslint.org/docs/developer-guide/working-with-rules#contextreport